identity/src/main/java/com/android/identity/credential/Credential.java

```diff
@@ -140,7 +147,7 @@ static Credential create(@NonNull StorageEngine storageEngine,
         return credential;
     }
 
-    private void saveCredential() {
+    private Boolean saveCredential() {
```

Comment:
I don't think there's any need to return a boolean here especially since we always seem to be returning `true`. Also, if you did you should use the primitive type `boolean` and not the boxed type `Boolean`. The latter is an instance of the class `java.lang.Boolean` and thus a lot heavier to dance with than the primitive type.